Src/DependencyCollector/Shared/Implementation/ClientServerDependencyTracker.cs

```diff
@@ -44,8 +44,6 @@ internal static DependencyTelemetry BeginTracking(TelemetryClient telemetryClien
 
             activity.Stop();
 #else
-            telemetryClient.Initialize(telemetry);
-
             // telemetry is initialized by Base SDK OperationCorrealtionTelemetryInitializer
```

Comment:
Yes, it's below,  but it looked like the comment implies that the Initialization you removed is still executed.